pkg/appfile/service_test.go

```diff
@@ -0,0 +1 @@
+package appfile
```

Comment:
delete this empty file?

Reply:
I will add tests in the following PRs here.